krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
We definitely mustn't do that! It'll cause the OS to do waaaaaaaay too much work. All we need is something like:
```python
j = 0
for i in range(10000000):
   j += 1
assert j == 10000000
```
and we should be OK on PyPy too (at least for the time being... in theory, if the trace optimiser becomes clever enough, it could even turn that into a constant, but let's not worry about that).